node/pkg/watchers/near/finalizer.go

```diff
@@ -20,7 +20,7 @@ type Finalizer struct {
 }
 
 func newFinalizer(eventChan chan eventType, nearAPI nearapi.NearApi, mainnet bool) Finalizer {
-	finalizedBlocksCache, _ := lru.New(workerCountTxProcessing * quequeSize)
+	finalizedBlocksCache, _ := lru.New(100_000)
```

Comment:
Why is this changing in this PR?

Reply:
Good catch. That shouldn't be in this PR. Will remove it.